domains/aux-domains.csv

```diff
@@ -5,4 +5,9 @@ region,short_name,long_name,nlon,nlat,ll_lon,ll_lat,dlon,dlat,pollon,pollat
 4,GAR-11,Greater Alpine Region,145,129,-14.955,-12.155,0.11,0.11,-162.0,39.25
 4,EUN-0275,EUCP Northern Europe,649,811,-9.99125,-0.20625,0.0275,0.0275,-162.0,39.25
 4,EUC-0275,EUCP Central Europe,361,401,-9.52375,-5.29375,0.0275,0.0275,-162.0,39.25
-4,SGAR-0275,Greater Alpine Region,289,271,-9.38625,-9.66625,0.0275,0.0275,-162.0,39.25
\ No newline at end of file
+4,SGAR-0275,Greater Alpine Region,289,271,-9.38625,-9.66625,0.0275,0.0275,-162.0,39.25
+4,GER-11,German Region,121,121,-10.89,-6.89,0.11,0.11,-162.0,39.25
+,WAF-11,Westafrican Region,361,271,-20.68,2.64,0.11,0.11,180.0,90.0
+,MSEA-11,Mainland South East Asia,145,181,92.34,7.12,0.11,0.11,180.0,80.0
+,MED-11,Mediterranean Region,487,321,-27.925,-25.925,0.11,0.11,198.0,39.25
+,CAS-11,Central Asian Region,257,271,-20.025,-20.605,0.11,0.11,-103.39,43.48
```

Comment:
I see here a conflict to the original CORDEX `CAS-11` (https://github.com/remo-rcm/tables/blob/main/domains/cordex-domains.csv#L11). Is that a small subset, e.g., focusing some country?

Reply:
![grafik](https://user-images.githubusercontent.com/5659125/220112397-f6e82b08-d967-4ad3-9126-b9aece05f523.png)

Reply:
@larsbuntemeyer, yes it is a subset of the original CORDEX CAS-11, we adjusted for our irrigation analysis, maybe we can call it CASi-11, if calling it CAS-11 is a problem

Reply:
good idea

Reply:
This is now handled in #4